skdownscale/pointwise_models/gard.py

```diff
@@ -74,11 +87,21 @@ class AnalogRegression(AnalogBase):
         KDTree object
```

Comment:
Its probably a good idea to add a note here like this:

```
Notes
-----
GARD models store prediction error  (`model.prediction_error_`) and 
exceedance probabilities (`model.exceedance_prob_`) for the last prediction. 
```

Reply:
same comment for the PureAnalog method

Reply:
done